shell/platform/embedder/tests/embedder_test_context.h

```diff
@@ -70,10 +72,7 @@ class EmbedderTestContext {
   friend class EmbedderConfigBuilder;
 
   std::string assets_path_;
-  std::unique_ptr<fml::Mapping> vm_snapshot_data_;
-  std::unique_ptr<fml::Mapping> vm_snapshot_instructions_;
-  std::unique_ptr<fml::Mapping> isolate_snapshot_data_;
-  std::unique_ptr<fml::Mapping> isolate_snapshot_instructions_;
+  std::string snapshot_path_;
```

Comment:
Same concern about this not being descriptive enough of an ivar name.

Reply:
Done.